README.md

```diff
@@ -2,8 +2,29 @@
 
 ## Description:
  
-To better support Instructors of Large Courses in Canvas, When we are given a CourseId we should be able create groups in Canvas based on their course sections. This will allow them to: 
-1) Share select resources with a section 
-2) Discussions available to specific section 
-3) Content Pages with a specific section 
-4) Conference with a specific section 
+To better support Instructors of Large Courses in Canvas,  given a CourseId we should be able create groups in Canvas based on their course sections. This will allow them to: 
+
+1. Share select resources with a section 
+2. Discussions available to specific section 
+3. Content Pages with a specific section 
+4. Conference with a specific section 
+
+This utility is written in python and takes `:course_id`, `:group_category_name` from the property file and creates a group category with the name provided in the property file. We will get all the sections and users
+in the course and create one group for each section and add all the users in each section to the corresponding group. The group name will be identical to the section name
+ 
+## Requirements 
+
```

Comment:
Along the lines of Composer for PHP, I'm thinking we should probably install the packages in user space (or the application's own installation directory, if that's possible) instead of site-packages.